### DISCUSSION TITLE
items sortable or searchable

### DISCUSSION THREAD

**Discussion by DarkDau** [linuxserver/Heimdall#1160] [Ideas]
If you have several entries in the application list, it is difficult to find a specific entry when you want to edit it.
It should not be a problem to sort the entries by name or date. Alternatively, a search function would be conceivable.

**Comment by KodeStar** [comment]
Does the "Tiles" search option not work for that?

**Reply by DarkDau** [reply]
can you show me where that is supposed to be?
